### ISSUE TITLE
Screenshot Drops - If Drops are more than 42, it will miss 1 line of the drop

### ISSUE THREAD

**Issue by nFeyz** [MathewSachin/Fate-Grand-Automata#430]
The apps only take the screenshot top of the drop page and end of the drop page, but if the drops are more than 42 drops (JP 90+ node), it will miss 1 line of the materials. Just want to let you guys know of this (will undercount if we submit the drop from the screenshot).
![18-9-2020-09-31-37 0](https://user-images.githubusercontent.com/53361797/93657561-5d786180-fa66-11ea-9954-d8b2f64b74b4.png)
![18-9-2020-09-31-37 1](https://user-images.githubusercontent.com/53361797/93657568-65380600-fa66-11ea-8b00-bd9d9edb2cee.png)

**Comment by MathewSachin** [comment]
Yeah, I know this is the case.
When the scrollbar isn't shown, we take 1 screenshot. If the scrollbar is there, we scroll to bottom and take one more.
For, the 3 screenshot case, detecting that there are 3 screenshots worth of drops is difficult

**Comment by nFeyz** [comment]
What if the apps scrolls the drops by dragging the materials? (like how you scroll the support) and detect if its at the end of the page or no by matching the end of the bar? 
![image](https://user-images.githubusercontent.com/53361797/93659023-7d625200-fa73-11ea-9197-b7ef943b9ac0.png)
![image](https://user-images.githubusercontent.com/53361797/93659041-b8648580-fa73-11ea-859d-9b68ce7e80e2.png)

**Comment by MathewSachin** [comment]
Yeah, that was what I was thinking. But, scrolling in this app isn't very precise, so there's no avoiding getting duplicate screenshots. I mean some of the drops will be shared between the screenshots since it's not possible to scroll exactly one page everytime.
